content/en/docs/_index.md

```diff
@@ -43,3 +43,21 @@ To build a bootable image is as simple as running `docker build`.
 - Cryptographically verified
 - instant switch from different versions
 - recovery mechanism with `cOS` vanilla images (or bring your own)
+
+## Mission
+
+The cOS-toolkit project is under the Elemental umbrella.
+
+Elemental provides a unique container based approach to define the system lifecycle of an immutable Linux derivative, without any string attached to a specific Linux distribution.
+
+At its heart, Elemental is the abstraction layer between Linux distro management and the specific purpose of the OS.
+
+Elemental empowers anyone to create derivatives from standard OCI images. Frees whoever wants to create a Linux derivative from handling the heavy bits of packaging and managing entire repositories to propagate upgrades, simplifying the entire process by using container images as base for OS.
+At the same time, Elemental provides an highly integrated ecosystem which is designed to be container-first, cloud native, and immutable.
+Anyone can tweak Elemental derivatives from the bottom-up to enable and disable its featureset.
+
+As the Elemental team, the [os2](https://github.com/rancher-sandbox/os2) project is our point of reference.
+
+`os2` is a complete derivative built with Elemental tied with the rancher ecosystem and full cycle node management solution with Kubernetes, which we are supporting directly and indirectly within changes also in the Elemental ecosystem.
+
+os2 is our main show-case, and as the Elemental team we are committed to it. It is also a show-off of how Elemental can be used to build and maintain immutable Linux derivatives: it encompasses several technologies to create a fully-automated, Kubernetes-focused Linux derivative which lifecycle is managed entirely from Kubernetes itself, [SDO](https://www.intel.it/content/www/it/it/internet-of-things/secure-device-onboard.html) included.
```

Comment:
fully automated? I did not get that from our talk the other day?

Reply:
good point! I think I worded poorly here, I wanted to underline fully automated installation, but I phrased it too generic

Reply:
updated in https://github.com/rancher-sandbox/cos-toolkit-docs/pull/72/commits/507fb0be8b2eadbc9f935cb8482365d16a916978